src/Algolia.Search/Models/Rules/Rule.cs

```diff
@@ -40,6 +40,11 @@ public class Rule
         /// </summary>
         public Condition Condition { get; set; }
```

Comment:
Can you mark this one as deprecated?

Reply:
Good point, thank you for reminding me. Doing it right now for C# and Go PRs.